src/applications/claims-status/components/claim-document-request-pages/DefaultPage.jsx

```diff
@@ -19,35 +19,86 @@ export default function DefaultPage({
   progress,
   uploading,
 }) {
+  const formattedDueDate = buildDateFormatter()(item.suspenseDate);
+  const formattedRequestedDate = buildDateFormatter()(item.requestedDate);
```

Comment:
Maybe we just build the formatter once and then call it twice? (I didn't get to the function definition yet, it's probably not terribly complex, but just in the interest of reuse)